MetaMorpheus/EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -275,18 +276,22 @@ public static void PepQValueInverted(List<SpectralMatch> psms, bool peptideLevel
 
         public void Compute_PEPValue(FdrAnalysisResults myAnalysisResults, List<SpectralMatch> psms)
         {
-            if (psms[0].DigestionParams.Protease.Name == "top-down")
+            string searchType;
```

Comment:
please add a comment about handling mixed data (top-down and bottom-up) not handled